pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1265,26 +1281,54 @@ func (og *operationGenerator) GenerateExpandVolumeFunc(
 
 	volumePlugin, err := og.volumePluginMgr.FindExpandablePluginBySpec(volumeSpec)
 
-	if err != nil {
+	if err != nil && volumeSpec.PersistentVolume.Spec.FlexVolume == nil {
```

Comment:
Is there a way we can abstract Flexvolume type here. We should not bring knowledge of Flexvolume into operationexcutor